airflow/config_templates/config.yml

```diff
@@ -2237,6 +2237,16 @@ scheduler:
       type: string
       example: ~
       default: "15"
+    default_cron_timetable:
+      description: |
+        Which Timetable to use when a cron string is provided to `schedule` argument of
+        a DAG. CronDataIntervalTimetable is the legacy Airflow behavior suitable for
+        DAGs with well-defined data_interval. CronTriggerTimetable is closer to the
+        behavior of cron itself.
+      version_added: 2.8.0
+      type: string
+      example: ~
+      default: "CronDataIntervalTimetable"
```

Comment:
Should this be a full importable string so the user may provide their own timetable implementation?

Reply:
I am unsure. I want to make this option as easy as possible to use because I think a _lot_ of Airflow users would rather use CronTriggerTimetable than the default. I almost wonder if it should just be a boolean to flip to trigger. Yes it is more restrictive, but if you're savvy enough to make your own Timetable, importing it in the DAG doesn't seem that hard.

Trying to think about what's best for the beginner, as the power user already has a lot of tools.

Reply:
If only the two timetables are allowed, I’d want to hide the timetables entirely and just have a boolean e.g. `legacy_cron_data_intervals` to switch between the two

Reply:
Sure, I like that.